tests/functions/test_vision.py

```diff
@@ -115,3 +115,29 @@ def test_draw_bounding_boxes(self, reset_db: None) -> None:
         assert 'number of boxes and box colors must match' in str(exc_info.value).lower()
 
         # TODO: test font and font_size parameters in a system-independent way
+
+    def test_draw_segmentation_masks(self, reset_db: None) -> None:
+        skip_test_if_not_installed('transformers')
+        from pixeltable.functions.huggingface import detr_for_segmentation
+        from pixeltable.functions.vision import draw_segmentation_masks
+
+        # Use a sample image URL
+        sample_url = (
+            'https://raw.githubusercontent.com/pixeltable/pixeltable/release/docs/resources/images/000000000034.jpg'
```

Comment:
Why not use the predefined SAMPLE_IMAGE_URL as in the above test (i.e. what's the rationale for doing things differently)?